src/App.tsx

```diff
@@ -38,7 +38,10 @@ function App() {
 
   const handleSubmit = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.preventDefault();
-    console.log("Click");
+    if (!inputTask) {
+      alert("You can't leave inpute blank");
```

Comment:
You are using material ui so use some component from it to show error message and not alert.